### PR TITLE
add gateway XBTS.io

### PR DESCRIPTION
**Pull request by technologiespro**

### CODE REVIEW

**Comment by technologiespro** [comment]
Hello!
announcement english: https://bitsharestalk.org/index.php?topic=26813.0
announcement russian: https://bitsharestalk.org/index.php?topic=26799.0
about team: https://smartholdem.io

**Comment by technologiespro** [comment]
Yes, we use simple(dafault) api for beta deposit/withdraw and rudex methods for main gateway page

**Comment by sschiessl-bcp** [comment]
Happy to hear about your experience with BitShares!

I tried contacting you in telegram, is your nickname correct?

**Comment by technologiespro** [comment]
Hello! my telegram @TechnoL0g
or https://t.me/TechnoL0g
Yes its correct.

**Comment by sschiessl-bcp** [comment]
I'm sorry I haven't contacted you yet. There is some background discussion going on and we will get back to you once that is resolved.

File: app/components/Modal/WithdrawModalNew.jsx [hunk 1]
```diff
@@ -125,6 +125,7 @@ class WithdrawModalNew extends React.Component {
         let gateFee = 0;
 
         if (selectedGateway) {
+            console.log(selectedGateway);
```
Comment (startailcoon) [inline app/components/Modal/WithdrawModalNew.jsx]: Please clean up the code when submitting a PR

Comment (technologiespro) [reply app/components/Modal/WithdrawModalNew.jsx]: thx u very match

**Comment by svk31** [comment]
@startailcoon

**Comment by startailcoon** [comment]
Will check on this as soon as possible, will be in a few days.

**Comment by startailcoon** [comment]
I'm unable to pass this due to:

- [ ] Withdraw & Deposit page doesn't list any form when selecting XBTS (see screen)
- [ ] Deposit (beta) isn't implemented
- [ ] Withdraw (beta) isn't implemented

image

**Comment by technologiespro** [comment]
thx, now included for everyone: Access-Control-Allow-Origin

**Comment by technologiespro** [comment]
Deposit (beta) & Withdraw (beta) fixed
please review

deposit

**Comment by startailcoon** [comment]
Thanks, will review this tonight

File: app/components/DepositWithdraw/xbtsx/XbtsxWithdrawModal.jsx [hunk 2]
```diff
@@ -0,0 +1,869 @@
...
+
+    _checkMinAmount() {
+        const {withdraw_amount} = this.state;
+
+        if (withdraw_amount === null) return;
+        const lessThanMinimum =
+            withdraw_amount <
+            this.props.min_amount /
+                utils.get_asset_precision(this.props.asset_precision);
+        console.log(
```
Comment (startailcoon) [inline app/components/DepositWithdraw/xbtsx/XbtsxWithdrawModal.jsx]: Please clean out unessesary console logs

**Pull request merged**